pyseir/inference/infer_rt.py

```diff
@@ -647,6 +669,7 @@ def infer_all(self, plot=True, shift_deaths=0):
 
             df["date"] = dates
             df = df.set_index("date")
+            df[f"{timeseries_type.value}"] = timeseries[start_idx:]
```

Comment:
Two things.

1. Can you add a comment why this is being added in with the given slicing on the timeseries. I can't easily reason about what indexes are being aligned here.

2. Why use the f-string instead of just `df[timeseries_type.value`?